code/datums/world_topic.dm

```diff
@@ -187,3 +187,6 @@
 
 	if(!key_valid)
 		GLOB.topic_status_cache = .
+
+/datum/world_topic/panic_bunker
```

Comment:
There's already the whitelist TGS command

Reply:
wow i did NOT know i'd saved that, whoops